journal/modernized/37-chapter-34.adoc

```diff
@@ -1394,6 +1414,8 @@ which they have learnt from their exterminating visitors.
 How long we may have to remain here, I know not;
 but I desire to be resigned to wait the blessed Master`'s time.
 
+--
+//I think this is where this letter ends, not certain though.
```

Comment:
Yup, I think you got it right here.  Maybe you're already doing this, but when you're in doubt, looking it up in the original often helps.  Sometimes in Wheeler it's _still_ not clear to me after looking at the original, but I find it usually helps to see how the original was formatted.

Reply:
Got it. Sometimes I forget we can look at the originals. :)